docs/configuration/command_line_interface.rst

```diff
@@ -4,3 +4,173 @@ Command Line Interface (CLI) commands
 #####################################
 
 .. vale on
+
+Mautic may require you to use the command line (CLI). Listed below are the available CLI commands.
+
+.. note:: 
+
+  You can find this list (and others - for example commands relating to Doctrine and other vendors) by running this command: ``bin/console``
+
+  You must be in the Mautic root directory to run the CLI commands. 
+
+**Format**: command [options] [arguments]
+
+Options
+=======
+
+.. vale off
+
+.. list-table:: 
+   :widths: 100 100
+   :header-rows: 1
+
+   * - Options
+     - Description
+   * - -h, \--help
+     - Display this help message
+   * - -q, \--quiet
+     - Do not output any message
+   * - | V, \--version
+       |  \--ansi
+       |  \--no-ansi
+     - | Display this app version
+       | Force ANSI output
+       | Disable ANSI output
+   * - -n, \--no-interaction
+     - 	Do not ask any interactive question
+   * - | -s, \--shell
+       |  \--process-isolation
+     - | Launch the shell.
+       | Launch commands from shell as a separate process.
+   * - | -e, \--env=ENV
+       |  \--no-debug
+     - | The Environment name. [default: "prod"]
+       |  Switches off debug mode.
+   * - | -v
+       | -vv
+       | -vvv
+       |  \--verbose
+     - | Increase the verbosity of messages:
+       | 1. for normal output,
+       | 2. for more verbose output and
+       | 3. for debug
+
+       
+.. vale on
+
+Mautic commands
+===============
+These are the commands you may need to use in relation to your Mautic instance. Add a ``bin/console`` before Mautic command.
+
+**Example**
+
+.. code-block:: shell
+
+  bin/console mautic:segments:update
+
+.. vale off
+
+.. list-table:: 
+   :widths: 100 100
+   :header-rows: 1
+
+   * - Command
+     - Description
+   * - mautic:assets:generate
+     - Combines and minifies Asset files from each bundle into single production files
+   * - mautic:broadcasts:send
+     - Process Contacts pending to receive a Channel broadcast.
+   * - mautic:campaigns:execute
+     - Execute specific scheduled events.
+   * - mautic:campaigns:messagequeue
+     - Process sending of messages queue.
+   * - mautic:campaigns:messages
+     - Process sending of messages queue.
+   * - mautic:campaigns:rebuild
+     - Rebuild Campaigns based on Contact Segments.
+   * - mautic:campaigns:trigger
+     - Trigger timed events for published Campaigns.
+   * - mautic:campaigns:update
+     - Rebuild Campaigns based on Contact Segments.
+   * - mautic:campaigns:validate
+     - Validate if a Contact has been inactive for a decision and execute events if so.
+   * - mautic:citrix:sync
+     - Synchronizes registrant information from Citrix products
+   * - mautic:cache:clear
+     - Clears the cache 
```

Comment:
This doesn't do the same thing as `cache:clear` command which clears the Symfony cache. This command clears the Mautic cache which is for example the dashboard widgets data or segment counts cached for 10 minutes. Mautic cache some data that are expensive to get live from the database or other sources (like API) for a short period of time. This cache is being cleared by this command. I hope you can explain it better than I can :D

Reply:
Thank you for the reviews @escopecz